package.json

```diff
@@ -1,8 +1,9 @@
 {
-  "name": "MMM-LKY-TIC",
+  "name": "mmm-lkx-tic",
```

Comment:
It would be 'mmm-lky-tic', lky standing for linky device

Reply:
Oh, sorry! I fixed it.